app/controllers/api/v1/emotions_controller.rb

```diff
@@ -13,6 +13,27 @@ def index
     end
   end
 
+  def create
+    @emotion = Emotion.new(emotion_params)
+    @emotion_existed = Emotion.all.find_by(word: params.dig('emotion', 'word'))
```

Comment:
@Maryna-Harasko you shouldn`t use instance variable (with @) if it used only in one method.